webui.lua

```diff
@@ -603,7 +625,8 @@ if passwd ~= 1 then
             .. concatkeys(servers, ':' .. options.port .. ' and ')
             .. ":" .. options.port
     )
-    message = function() mp.osd_message(MSG_PREFIX .. startup_msg, 5) end
+--    message = function() mp.osd_message(MSG_PREFIX .. startup_msg, 5) end
+    message = function() end
```

Comment:
Ok, you don't like the startup message :laughing: .

We could have an option to toggle them off. But that would be something for another PR. Please revert.

Reply:
sorry, didn't mean to include these local hacks in this PR